include/veriblock/pop/blockchain/blockchain_util.hpp

```diff
@@ -152,6 +152,21 @@ void payloadToCommands(Tree& tree,
 
 struct PopData;
 
+//! @private
+template <typename Tree, typename Payload>
+void payloadToCommandGroup(Tree& tree,
+                           const Payload& payload,
+                           const std::vector<uint8_t>& containingHash,
+                           CommandGroup& cg) {
+  // const auto& pl = tree.getPayloadsIndex();
+
+  cg.payload_type_name = &Payload::name();
+  cg.id = payload.getId().asVector();
+  // cg.valid = pl.getValidity(containingHash, cg.id);
```

Comment:
?

Reply:
the validity index is not functional ATM, so I disabled it to remove the associated overhead

Reply:
why not removed? why is it commented?

Reply:
it's potentially useful code + a reminder that we have CommandGroup::valid member